cmd/state-installer/installer.go

```diff
@@ -37,7 +37,7 @@ func main() {
 		if panics.HandlePanics(recover(), debug.Stack()) {
 			exitCode = 1
 		}
-		if err := events.WaitForEvents(1*time.Second, analytics.Wait, rollbar.Close, authentication.LegacyClose); err != nil {
```

Comment:
This PR switched to a non-global analytics client.  As it is not used anywhere in the `state-installer`, we can be certain that it will not be called anywhere.  So, we do not need to wait for it finish either.

Reply:
I think the use-case is still there, yes.  It is just not used right now...  Should I then initialize an `analytics.DefaultClient` anyways.  So, the installer is at least set up for it?